flocker/node/agents/test/test_blockdevice.py

```diff
@@ -1244,6 +1253,27 @@ def __init__(self, dataset_id, node_uuid, mountroot, blockdevice_manager):
         self._testing_mountroot = mountroot
         self._blockdevice_manager = blockdevice_manager
 
+    def _immitate_docker_writing(self, mountdir, filename, content):
+        """
+        Imitate writing content to a file to a mounted directory using docker.
+
+        :param FilePath mountidr: The directory to be mounted as a docker
+            volume.
+        :param unicode filename: The name of the file to create in the docker
+            volume.
+        :param unicode content: The content to write to the file.
+        """
+        container_path = FilePath('/vol')
```

Comment:
This seems rather magic to me. Is a comment as to why
we are removing that path then writing another warranted?